MetaMask/metamask-extension
# feat: update celo chain logo svg

## **Description**



Add updated Celo svg logo

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to add network
2. Add celo networks
3. celo networks should have updated logo

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
